perf/benchmark/runner/runner.py

```diff
@@ -324,6 +335,11 @@ def run(self, headers, conn, qps, size, duration):
 
         load_gen_cmd = ""
         if self.load_gen_type == "fortio":
+            if self.security_policy is not None:
+                security_cmd = "-security_policy={security}".format(security=self.security_policy)
```

Comment:
this `-security_policy` here is to call your generate policy code, well, we have the top level `--security_policy` to trigger the whole test flow. We'd better to rename the flag here `-security_policy` to differentiate from the top level `--security_policy`

Reply:
something like: `-generate_policy`

Reply:
updated to -generate_policy